data/json/items/comestibles/serums.json

```
@@ -0,0 +1,30 @@
+[
+  {
+    "id": "serum_regen",
+    "type": "COMESTIBLE",
+    "comestible_type": "MED",
+    "category": "drugs",
+    "name": { "str_sp": "unknown liquid" },
+    "description": "White, semi-transparent liquid of unknown origin.",
+    "weight": "7 g",
+    "volume": "10 ml",
+    "charges": 1,
+    "phase": "liquid",
+    "material": [ "alien_liquid" ],
+    "symbol": "[",
+    "color": "white",
+    "use_action": [
+      {
+        "type": "effect_on_conditions",
+        "menu_text": "Inject",
+        "description": "Inject the liquid.",
+        "effect_on_conditions": [
+          {
+            "id": "EOC_RUN_SERUM_REGEN",
+            "effect": { "run_eoc_with": "EOC_SERUM_GENERIFIED", "variables": { "eoc": "EOC_SERUM_REGEN" } }
```

```suggestion
            "effect": { "run_eoc_with": "EOC_SERUM_GENERIFIED", "variables": { "serum_duration": "86400" } }
```
Let's see if these can help solving the error.

---

duration error happened because i accidentally mixed the names of context vals
the fix was in my local, i just forgot to push it, now it should work